### PR TITLE
fix(pipeline): Solve the problem of automatically opening the timing when on push

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
Solve the problem of automatically opening the timing when on push

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb25JRHMiOlsxMjgwLDEyOTddLCJhc3NpZ25lZSI6WyIxMDAxMjYxIl19&id=319317&iterationID=1280&pId=0&type=BUG)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      Solve the problem of automatically opening the timing when on push        |
| 🇨🇳 中文    |       解决on push的时候自动开启定时问题       |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sfwn** [comment]
rebase master please

File: internal/tools/pipeline/providers/cron/cron.service.go [hunk 1]
```diff
@@ -122,8 +119,7 @@ func (s *provider) CronCreate(ctx context.Context, req *pb.CronCreateRequest) (*
 				return err
 			}
 		}
-
-		return nil
+		return s.addIntoPipelineCrond(createCron)
```
Comment (sfwn) [inline internal/tools/pipeline/providers/cron/cron.service.go:122]: why move to the function end

File: internal/tools/pipeline/providers/cron/cron.service_test.go [hunk 2]
```diff
@@ -669,3 +669,82 @@ func Test_provider_update(t *testing.T) {
...
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			s := &provider{
+				Daemon: tt.fields.Daemon,
+			}
+			if err := s.addIntoPipelineCrond(tt.args.cron); (err != nil) != tt.wantErr {
+				t.Errorf("addIntoPipelineCrond() error = %v, wantErr %v", err, tt.wantErr)
```
Comment (sfwn) [inline internal/tools/pipeline/providers/cron/cron.service_test.go:746]: assert error is meaningless

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/tools/pipeline/providers/cron/cron.service_test.go (hunk 2 above)
Comment (littlejiancc) [reply internal/tools/pipeline/providers/cron/cron.service_test.go:749]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**